README.md

```diff
@@ -1,8 +1,151 @@
+
+# Getting Started
+
+While this section primarily reiterates [Scroll's Devnet Guide](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/), it includes adjustments specifically for Ubuntu environments.
+This guide also makes revisions to launch the coordinator service which acts as a gateway between the sequencing layer and Sindri provers.
+Since the coordinator requires at least 20Gb of RAM, we recommend you use a machine with at least 32Gb available.
+
+### Prerequisites
+
+You will need to obtain an API key to use Sindri.
+If you have not already created an account, follow the sign up link posted at the top left of this README.
+After logging into the [Sindri front-end](https://sindri.app/login), you can create and manage your API Keys within the [API Keys Settings page](https://sindri.app/z/me/page/settings/api-keys).
+
+You should also install `docker`, `kubectl`, `minikube`, `helm`, `node`, and `scrollsdk` as instructed by the official [Scroll SDK documentation](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/#prerequisites).
+
+You should initialize your `minikube` environment via
+```
+minikube start --cpus=12 --memory=32768
+minikube addons enable ingress
+minikube addons enable ingress-dns
+```
+In the first command, you are allowing the minikube container to use up to 32Gb of RAM.
+The following two commands enable external traffic to reach the minikube cluster so that you can interact with the chain outside the minikube container.
+
+### Obtaining and Configuring Charts
+
+You first need to clone the Scroll SDK repo and navigate to the `devnet/` directory to access some helper scripts.
+```bash
+git clone git@github.com:scroll-tech/scroll-sdk.git
+cd scroll-sdk/devnet
+```
+
+This next command manually pulls and extracts the charts for the latest version of Scroll SDK.
+```bash
+make bootstrap
+```
+> ⚠️ **Encountering Permission Issues?**<br>
+> If you are given any prompts during the execution of the bootstrap command or if `make install` does not successfully execute, then there is likely a root vs. user mismatch between configuration shell scripts and the downloaded files.
+> You can solve this by exiting the bootstrap process with `ctrl+c` and running the `config` command separately with `sudo make config`.
+
+
+### Installing the Helm Chart
+
+In this step, we will launch the sequencing layer, the coordinator, and various visibility services.
+Because provers require manual configuration, we will start those separately.
+Note that the standard devnet settings do not include any proving layer services, so you will need to add the following two lines to the end of the `install` command in `scroll-sdk/devnet/Makefile`.
+```bash
+    --set coordinator-api.enabled=true \
+    --set coordinator-cron.enabled=true
+```
+After that adjustment, enter `make install` in your terminal to start the chain.
```

Comment:
Starting the chain seems like a major step, and yet `make install` is buried in the sentence. I'd prefer showing a full code block like the earlier examples.

Reply:
Commit https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/8/commits/9afb66b07b940a4a41c20bc93f6c915c9e34cfef added a dedicated codeblock and while I was there, it seemed like some extra "what to look for after running this command" was called for.